<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>小公主连连看</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gamePanel" native="true">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>191</width>
      <height>261</height>
     </rect>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>111</width>
       <height>241</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="startBtn">
        <property name="text">
         <string>进入游戏</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="rankingBtn">
        <property name="text">
         <string>排行榜</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="gameIntronduceBtn">
        <property name="text">
         <string>游戏介绍</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="gameHelpBtn">
        <property name="text">
         <string>游戏帮助</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="gameSettingBtn">
        <property name="text">
         <string>游戏设置</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="gameExitBtn">
        <property name="text">
         <string>退出游戏</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>0</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>欢迎您，陌意随影</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
